lib/Backend/GlobOpt.cpp

```diff
@@ -6935,20 +6959,34 @@ GlobOpt::SetSymStore(ValueInfo *valueInfo, Sym *sym)
     }
     if (valueInfo->GetSymStore() == nullptr || valueInfo->GetSymStore()->IsPropertySym())
     {
-        valueInfo->SetSymStore(sym);
+        SetSymStoreDirect(valueInfo, sym);
     }
 
     return sym;
 }
 
 void
+GlobOpt::SetSymStoreDirect(ValueInfo * valueInfo, Sym * sym)
```

Comment:
Oh I get it.  I miss the fact there already was a SetSymStore on GLobOpt.

---
In reply to: [68327517](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r68327517) [](ancestors = 68327517,68327056)